src/components/cards/NewFeatureCard.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <timed-card startTime="March 6, 2025" endTime="March 8, 2025">
+  <timed-card startTime="May 8, 2025" endTime="May 26, 2025">
     <div class="container">
       <div class="title"></div>
       <div class="message">
-        <a class="link" href="/snowblitz" target="_blank">❄️ Operation Snowball's Snowblitz is TODAY!</a> Don't miss out on a night of fun activities, team building, bonding with friends, and pizza! 
+        SHS's 2024-25 Music Production 2 classes engaged in an activist songwriting project centered around important topics to raise awareness for
```

Comment:
Potentially a verbal callout & link to Jukebox? That's how we should be connecting this back to make it seem less pluggy.

Reply:
Jukebox should also have just a small hyperlink to the playlist.

Reply:
Perfect! Callout can be as simple as "Listen to more student-made music here" (bit long, but you get the point haha)